lib/FrontendTool/FrontendTool.cpp

```diff
@@ -412,8 +412,8 @@ static bool emitLoadedModuleTraceIfNeeded(ModuleDecl *mainModule,
   mainModule->getImportedModules(imports, filter);
 
   SmallPtrSet<ModuleDecl *, 8> importedModules;
-  for (std::pair<ModuleDecl::AccessPathTy, ModuleDecl *> &import : imports)
-    importedModules.insert(import.second);
+  for (auto &import : imports)
```

Comment:
Nitpick: Since the type was originally written explicitly (presumably for readability/clarity), please don't swap it out for `auto`.

Reply:
Got it. I'll push up another commit to fix this.

Reply:
Resolved in 1d625d8